app/elements/devguide-shell.html

```diff
@@ -216,17 +277,39 @@
     },
 
     attached: function() {
+      Polymer.RenderStatus.afterNextRender(this, function() {
+        // Somebody might look at the API docs. Be ready.
+        this.importHref(this.resolveUrl('/elements/iron-doc-viewer.html'));
+      });
+
       // Compute the active page from the parent pw-shell's app route.
       var pwShell = this.parentElement;
```

Comment:
Should use `Polymer.dom(this).parentElement` - this works right now since `pw-shell` has `<content>` at the top level, but that's an implementation detail that shouldn't be relied upon.